core/src/lib.rs

```diff
@@ -72,10 +83,7 @@ impl Plugin for CorePlugin {
             .register_type::<RotationConstraints>()
             .register_type::<CollisionLayers>()
             .register_type::<SensorShape>()
-            .add_system_to_stage(CoreStage::First, PhysicsSteps::update.system())
-            .add_stage_before(CoreStage::PostUpdate, crate::stage::ROOT, {
-                Schedule::default().with_stage(crate::stage::UPDATE, SystemStage::parallel())
-            });
```

Comment:
I moved this to https://github.com/JRMurr/heron/blob/custom-stages/src/lib.rs#L161. So its only added in the "normal" plugin

Reply:
After the discussed changes to not break the existing api, all the "old" plugins will check if this stage does not exist and try to add it. This way if a user used any of the old plugins the stage would be registered as expected